lib/article_json/export/apple_news/elements/text_box.rb

```diff
@@ -0,0 +1,56 @@
+module ArticleJSON
+  module Export
+    module AppleNews
+      module Elements
+        class TextBox < Base
+          include ArticleJSON::Export::Common::HTML::Elements::TextBox
+          # List
+          # @return [Hash]
+          def export
+            {
+              role: 'container',
+              layout: 'textBoxLayout',
+              style: 'textBoxStyle',
+              animation: {
+                type: 'appear',
+                userControllable: true,
+                initialAlpha: 0.0,
+              },
+              components: map_styles(elements)
+            }
+          end
+
+          private
+
+          # @return [Array]
+          def elements
+            @element.content.map do |child_element|
+              klass = child_element.class
+              if klass == ArticleJSON::Elements::Heading
+                namespace::Heading.new(child_element).export
+              elsif klass == ArticleJSON::Elements::Paragraph
+                namespace::Paragraph.new(child_element).export
+              elsif klass == ArticleJSON::Elements::List
+                namespace::List.new(child_element).export
+              else
+                namespace::Text.new(child_element).export
+              end
```

Comment:
```suggestion
              case klass
              when ArticleJSON::Elements::Heading
                namespace::Heading.new(child_element).export
              when ArticleJSON::Elements::Paragraph
                namespace::Paragraph.new(child_element).export
              when ArticleJSON::Elements::List
                namespace::List.new(child_element).export
              else
                namespace::Text.new(child_element).export
              end
```

Reply:
Through a quick pairing we noted that `if` and `case` statements are not an exact like for like if you use `==` like I did here. `case` statements actually use `===` for comparison.